Source/Resource/Plugin/Language/eu.csv

```diff
@@ -1,6 +1,6 @@
   Language_ISO639;eu
-  Author_Email;azpidatziak@gmail.com
-  Author_Name;Xabier Aramendi

```

Comment:
Put the older name in `Author_OldNames` field.

Reply:
I forgot it. I will do it.